lib/git-process.ts

```diff
@@ -34,7 +34,7 @@ export interface IGitSpawnExecutionOptions {
    * set as environment variables before executing the git
    * process.
    */
-  readonly env?: Object
+  readonly env?: object
```

Comment:
`object` is the correct type to use 99% of the time; using `Object` creates problems because e.g. a deep-partial regular object can't be an `Object` but is a valid `object`.